Lombiq.ContentEditors/Assets/Scripts/async-editor/async-editor.js

```diff
@@ -226,12 +227,12 @@ window.asyncEditor.editor = {
             const index = editorGroups.indexOf(editorGroup ?? this.editorGroup);
 
             return editorGroups[index + 1];
-        }
+        },
     },
 };
 
 window.initAsyncEditor = (asyncEditorId, parameters) => {
-    if (!parameters) return window.asyncEditor.editors[asyncEditorId];
+    if (!parameters) return;

```

Comment:
Why is this change necessary?

Reply:
To fix an eslint error because here we were returning something, but if there are parameters, below we init `window.asyncEditor.editors[asyncEditorId]` but nothing is returned.

Reply:
OK then.